handbook/ops/bizops/ab-testing.md

```diff
@@ -0,0 +1,76 @@
+
+A/B tests take time and resources, and we can only run a few A/B tests at a time to avoid interferences between tests. We should keep A/B testing for high-impact metrics optimization, that match the other conditions here.
+
+#### Uncertainty
+
+A/B testing is useful when there are several options to choose from, possibly with competing explanations on why they would work, but no clear winner.
+
+It’s no use A/B testing versions if we already know the result with high probability. Sometimes though, we will want to use A/B testing for things we know are important just to *quantify* impact (is it worth maintaining this?), as opposed to just *validate* (yes/no) impact.
+
+#### Measurement
+
+A/B testing requires a clear quantitative way to measure what is "better". A common advise is to pick a single, clear metric so that there is no ambiguity in deciding if the change `passed` or `failed`.
+
+
+#### Volume of users to reach statistical significance
+
+A/B testing requires a large enough number of users to get statistically significant results. The smaller the number of users in the A/B testing cohort, the more the change needs to have a big impact on the metric to be measurable. This is because the signal (the impact of the change) needs to be higher than the noise (the random fluctuation in a metric).
+
+For example, retention fluctuates week over week for "random" reasons we don't understand. The more users, the smaller those "random" fluctuations will be: this will make it possible to decide wether a change caused by an experiment was likely random, or likely caused by the experiment.
+
+Running A/B tests on a number of users that is too small is inefficient (waste time to setup the A/B test), inconclusive (the result will be that no conclusion can be made) and sometimes counterproductive (miscommunicated, an insignificant A/B test leads teammates to draw conclusions that aren't valid and can be hard to debunk). In that case, it's better to choose another method, such as user interviews.
+
+### Examples
+
+#### Good A/B test candidates
+
+- Evaluating signup flows, or any flow that result in a measurable action
+- Evaluating versions of button placement, CTAs, landing pages
+
+#### Bad A/B test candidates
+
+- Launching a new key feature backed by product and UX research: success can be measured directly with an adoption metric (there is an easier way)
+- Fixing a frequently reported bug likely does not require A/B testing (no uncertainty)
+- Changing the design to build brand trust (no metric)
+
+## How to run an A/B test
+
+After you identify a good A/B test candidate:
+
+### Define the A/B test
+
+- Define the target metric
+- Define the A, B (and more) versions
+- Define the length of the test, depending on the number of users you need, and check for statistical significance with a calculator ([example](https://www.optimizely.com/sample-size-calculator/))
+- Pick a feature flag name for your A/B test (eg. `w0-signup-optimization`)
```

Comment:
```suggestion
- Pick a feature flag name for your A/B test (eg. `w0-signup-optimisation`)
```
It's optimisation in all our code/data so want to make sure people can find it if needed

Reply:
Nice. My autocorrect will like this.